core-bundle/src/Resources/contao/dca/tl_image_size.php

```diff
@@ -378,24 +378,59 @@ public function editHeader($row, $href, $label, $title, $icon, $attributes)
 	 */
 	public function getFormats()
 	{
-		$options = array();
-
-		if (\in_array('webp', StringUtil::trimsplit(',', \Config::get('validImageTypes'))))
+		if (!\in_array('webp', StringUtil::trimsplit(',', \Config::get('validImageTypes'))))
 		{
-			$options[] = 'png:webp,png';
-			$options[] = 'jpg:webp,jpg;jpeg:webp,jpeg';
-			$options[] = 'gif:webp,gif';
+			$GLOBALS['TL_DCA']['tl_image_size']['fields']['formats']['label'] = array
+			(
+				$GLOBALS['TL_LANG']['tl_image_size']['formats'][0],
+				$GLOBALS['TL_LANG']['tl_image_size']['formatsMissingWebp'],
```

Comment:
I don't think that we still need this, do we? If the admin has explicitly disabled WEBP, they might also have disabled the input field. And even if not, they might not want us to tell the users to discuss their decision with them. 😄

Reply:
Done 7a21f88c770d7fdd0c68ff0c6bdee5d0ef0a9dc5